Package.swift

```diff
@@ -36,7 +36,7 @@ let package = Package(
         ),
         .package(
             url: "https://github.com/groue/GRDB.swift",
-            exact: "6.16.0"
+            exact: "6.29.1"
```

Comment:
We should update this to allow for a range of versions. Would start small with anything in `6.29.X`. 

This does open up the possibility of bugs occurring on specific versions of GRDB, so we may want to look into retrieving the finalized version when an application is built